maintenance_equipment_scrap/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Maintenance Equipments Scrap',
+    'summary': 'Enhance the functionality for Scrapping Equipments',
+    'author': 'Onestein, Odoo Community Association (OCA)',
+    'website': 'http://www.onestein.eu',
+    'category': 'Human Resources',
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'maintenance',
+        'mail',
+        'stock',
```

Comment:
I can see that you added the Stock dependency to be able to add a Settings option.
You thought well, but I think that this is a too heavy dependency.
I would prefer to have it removed, even if that also means dropping the Settings option for now.
ideally, we could have a "maintenance_settings" module, providing the missing Settings menu for the Maintenance app, and then we could add this option there.
What do you think?